android/app/src/main/kotlin/com/chicagoroboto/ext/ViewExt.kt

```diff
@@ -0,0 +1,6 @@
+package com.chicagoroboto.ext
+
+import android.app.Activity
+import android.view.View
+
+fun View.getActivity(): Activity? = if (context is Activity) context as Activity? else null
```

Comment:
random thought and probably not worth the change, but maybe unwrap ContextWrappers? Only matters for apps use `android:theme` in xml or build other deep scopes. shrug.
```kotlin
fun View.getActivity(): Activity {
  var context = context
  while (context !is Activity) {
    if (context is ContextWrapper) {
      context = context.baseContext
    } else {
      throw RuntimeException("Could not find Activity.") // or return null.
    }
  }
  return context
}
```